pkg/components/gtime/gtime_test.go

```diff
@@ -14,25 +15,26 @@ func TestParseInterval(t *testing.T) {
 	tcs := []struct {
 		interval string
 		duration time.Duration
-		err      string
+		err      *regexp.Regexp
 	}{
 		{interval: "1d", duration: now.Sub(now.AddDate(0, 0, -1))},
 		{interval: "1w", duration: now.Sub(now.AddDate(0, 0, -7))},
 		{interval: "2w", duration: now.Sub(now.AddDate(0, 0, -14))},
 		{interval: "1M", duration: now.Sub(now.AddDate(0, -1, 0))},
 		{interval: "1y", duration: now.Sub(now.AddDate(-1, 0, 0))},
 		{interval: "5y", duration: now.Sub(now.AddDate(-5, 0, 0))},
-		{interval: "invalid-duration", err: "time: invalid duration invalid-duration"},
+		{interval: "invalid-duration", err: regexp.MustCompile(`^time: invalid duration \"invalid-duration\"$`)},
```

Comment:
I would expect this to fail given [the error starts with `ParseInterval` and the input interval is not quoted](https://github.com/grafana/grafana/blob/b2a08c2a53f8ce038a819662d08549c6738e9082/pkg/components/gtime/gtime.go#L35).

Reply:
@papagian I don't know why Go 1.15 is different, but it has quotes. I'm in the middle of preparing a fix that uses a regexp.

Reply:
Should be fixed now.

Reply:
I noticed this in the go 1.15 release notes. It is actually the very last thing :-P https://golang.org/doc/go1.15#time 

> When returning an error, ParseDuration now quotes the original value.

Reply:
Thanks @kylebrandt for clearing that up :+1: I've fixed it using regexps.

Reply:
So in-fact this error is returned [here](https://github.com/grafana/grafana/blob/b2a08c2a53f8ce038a819662d08549c6738e9082/pkg/components/gtime/gtime.go#L17).